.github/workflows/dockerimage.yml

```diff
@@ -44,13 +44,13 @@ jobs:
         fi
         echo "::set-env name=HUB_TAG::${DOCKER_HUB_BASE_NAME}:${TAG_NAME}"
 
-    - name: Pull the Docker image
-      run: |
-        docker pull "${HUB_TAG}"
-
     - name: Build the Docker image
       run: |
-        docker build --cache-from "${HUB_TAG}" . --build-arg PYTHON_VERSION=${{ matrix.python_version }} --build-arg BUILD_TYPE=${{ matrix.build_type }} --file Dockerfile --tag "${HUB_TAG}"
+        if [ "${{ github.event_name }}" != 'release' ]; then
```

Comment:
How about including an explanatory comment about why this condition is required and that we'll get a "cache miss" on new releases?

Reply:
That makes sense. I added a comment in https://github.com/optuna/optuna/pull/1791/commits/c3fc8db6477906dfc49a01e4c06a748e7c1307bd.